.goreleaser.yml

```diff
@@ -68,4 +70,7 @@ brews:
     description: Algolia CLI utility
     install: |
       bin.install "algolia"
+      bash_completion.install "completions/algolia.bash" => "algolia"
```

Comment:
Maybe I'm missing something here but we install autocompletion for zsh bash and fish on every machine? maybe we could/should ask the user which one to choose with a prompt first?

Moreover, I don't find the meaning on `=>` and why it's used on fish install command 🤔

Reply:
The `bash_completion.install` and the following ones are provided by `HomeBrew` and will only be run in the appropriate context, depending of the type of shell the user is running, so it's safe to do the three (and the recommended way).

Reply:
Couldn't find the docs for this, but you can see examples of similar thing in the GitHub CLI and other formula: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/hub.rb#L35

Reply:
Awesome thanks 🙏